rules/windows/defense_evasion_windows_firewall_profile_disabled.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2021/10/15"
+maturity = "production"
+updated_date = "2021/10/15"
+
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies when a user disables the Windows Firewall Profile to help evade defense.
```

Comment:
Needs a better description, what is disabled is the windows Firewall, the profile is just the scope. using `-All` instead of a specific profile would result in the firewall to be disabled in every profile for example.

Reply:
++